test/e2e/util.go

```diff
@@ -761,14 +761,20 @@ func ReadLatencyMetrics(c *client.Client) ([]LatencyMetric, error) {
 
 // Prints summary metrics for request types with latency above threshold
 // and returns number of such request types.
-func HighLatencyRequests(c *client.Client, threshold time.Duration) (int, error) {
+func HighLatencyRequests(c *client.Client, threshold time.Duration, ignoredResources []string) (int, error) {
 	metrics, err := ReadLatencyMetrics(c)
 	if err != nil {
 		return 0, err
 	}
+	ignored := make(map[string]bool)
+	for _, resource := range ignoredResources {
```

Comment:
suggest declaring this as a util.StringSet and checking the set with Has, instead of a list that becomes a dict